workflows/ci-simple/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,166 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: argo-hub.ci-simple.0.0.1
+  annotations:
+    argo-hub/version: '0.0.1'
+    argo-hub/description: 'This Workflow Template is used to create your first CI pipeline in CSDP'
+    argo-hub/categories: 'argo'
+    argo-hub/license: 'MIT'
+    argo-hub/owner_name: 'Eti Zaguri'
+    argo-hub/owner_email: 'eti.zaguri@codefresh.io'
+    argo-hub/owner_avatar: 'https://avatars.githubusercontent.com/u/85868206'
+    argo-hub/owner_url: 'https://github.com/eti-codefresh'
+    argo-hub/icon_url: "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/workflows/starting-template/assets/icon.svg"
+    icon-background: "#171A2D"
+spec:
+  #   arguments:
+  # #     List here all the arguments to pass as variables to the Workflow from the Sensor
+  # #     Use this syntax to pass them into the Workflow Templates: "{{workflow.parameters.REPO}}"
+  # #     These are the arguments that are required for this CI pipeline:
+  #     parameters:
+  #       - name: REPO
+  #       - name: IMAGE_NAME
+  #       - name: TAG
+  #       - name: GIT_REVISION
+  #       - name: GIT_BRANCH
+  #       - name: GIT_COMMIT_URL
+  #       - name: GIT_COMMIT_MESSAGE
+  #       - name: DOCKERFILE
+  #       - name: GIT_TOKEN_SECRET
+  #       - name: CONTEXT
+  #       - name: REGISTRY_CREDS
+
+  # To use the build-docker-image step from the argo-hub.kaniko.0.0.1 template, you need to create a docker-registry secret
+  # Use your docker credentials for the runtime namespace
+  # https://jamesdefabia.github.io/docs/user-guide/kubectl/kubectl_create_secret_docker-registry/
+  volumes:
+    - name: docker-config
+      secret:
+        items:
+          - key: .dockerconfigjson
+            path: config.json
+        secretName: docker-registry # this is the name of the docker-registry secret you created
```

Comment:
For some secrets we ask for the secret name as input (GIT_TOKEN_SECRET, REGISTRY_CREDS), and for others we assume a standard name (docker-registry). I think we should do them all the same.